test/client/fixtures/sandbox/code-instrumentation/setters-test.js

```diff
@@ -134,6 +134,17 @@ test('anchor', function () {
         strictEqual(anchorWithNotSupportedProtocol.origin, nativeMethods.anchorOriginGetter.call(etalonAnchorWithNotSupportedProtocol));
 });
 
+test('image (GH-1502)', function () {
```

Comment:
I don't understand what this test do.

Reply:
It's partial duplication of getter test, so i delete it.